### PR TITLE
fix reloading page in auth component issue #5

### PR DESCRIPTION
**Pull request by aotten099**

### CODE REVIEW

File: client/src/app/authentication/auth.service.ts [hunk 1]
```diff
@@ -1,138 +1,90 @@
...
   }
 
-  login(redirectPath: string = '/') {
-    // A desired redirect path can be passed to login method
-    // (e.g., from a route guard)
-    // Ensure Auth0 client instance exists
-    this.auth0Client$.subscribe((client: Auth0Client) => {
-      // Call method to log in
-      client.loginWithRedirect({
+  login(redirectPath: string = '/home/dashboard'): Observable<void> {
```
Comment (ErikRauer) [inline client/src/app/authentication/auth.service.ts:61]: This doesn't look right to me. I believe the string should just be '/' for our project

**Comment by ErikRauer** [comment]
Also it would be nice to remove the lgtm alerts. To do so you would just have to remove the imports highlighted here:
[https://lgtm.com/projects/g/UMM-CSci-3601-S20/it-3-caymanislandsolympicathletes2016/rev/pr-abc476796b3d9738de985d5213eb2ff587357f32]

File: client/src/app/authentication/auth.service.ts (hunk 1 above)
Comment (aotten099) [reply client/src/app/authentication/auth.service.ts]: You're right, I missed that one.

**Comment by aotten099** [comment]
I closed the pull request for now, as i am going to look into the redirect that is causing me issues.

**Comment by audreylemeur** [comment]
Instead of closing it, I converted it to a draft until you feel like it's done

**Pull request merged**